src/openrct2/interface/console.c

```diff
@@ -454,19 +455,20 @@ static sint32 cc_rides(const utf8 **argv, sint32 argc)
 			FOR_ALL_RIDES(i, ride) {
 				char name[128];
 				format_string(name, 128, ride->name, &ride->name_arguments);
-				console_printf("rides %03d type: %02u subtype %03u name %s", i, ride->type, ride->subtype, name);
+				console_printf("ride: %03d type: %02u subtype %03u operating mode: %02u name: %s", i, ride->type, ride->subtype, ride->mode, name);
 			}
 		} else if (strcmp(argv[0], "set") == 0) {
 			if (argc < 4) {
 				console_printf("rides set type <ride id> <ride type>");
+				console_printf("rides set mode <ride id> <operating mode>");
```

Comment:
One way of providing user-facing list of modes would be changing this to read:
`console_printf("rides set mode [<ride id> <operating mode>]");`
and when not provided both numbers, simply listing modes.